### PR TITLE
feat: Improve speed by calculating point to point the peak shape.

### PR DESCRIPTION
**Pull request by jobo322**
remove the approximation of the shape values with a pre-computed shape.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -246,29 +234,16 @@
...
 }
 
-function createShape(kind, options) {
-  const hash = objectHash({ kind, options });
-  if (shapesCache[hash]) {
-    return shapesCache[hash];
+function createShape(kind) {
+  if (typeof kind === 'string') kind = getKind(kind);
+  switch (kind) {
+    case GAUSSIAN:
```
Comment (lpatiny) [inline src/index.js:240]: function is a little 'hot (reserved word)' maybe we could 'shapeFct'

**Comment by lpatiny** [comment]
Could you ahve a look at the README as well ?

File: src/index.js [hunk 2]
```diff
@@ -246,29 +234,16 @@
   });
 }
 
-function createShape(kind, options) {
-  const hash = objectHash({ kind, options });
-  if (shapesCache[hash]) {
-    return shapesCache[hash];
```
Comment (lpatiny) [inline src/index.js:252]: This should move to the shape project as 'getShapeFct'

File: src/index.js (hunk 1 above)
Comment (jobo322) [reply src/index.js:240]: the default factor is ok?

File: src/index.js (hunk 2 above)
Comment (jobo322) [reply src/index.js:252]: it is done

**Pull request merged**